test/regression/RegressionTest.hh

```diff
@@ -8,7 +8,18 @@
 #include "llvm/MC/MCAsmBackend.h"
 #include "llvm/MC/MCAsmInfo.h"
 #include "llvm/MC/MCCodeEmitter.h"
+
+#if defined(__clang__)
```

Comment:
This prevents warnings (that turn to errors and prevent compiling because of -Werror) in this specific llvm file when clang is being used. A slightly janky workaround but couldn't find anything better

Reply:
Added comments